### DISCUSSION TITLE
You Name It, We Do It!

### DISCUSSION THREAD

**Discussion by JohnGuan** [IceWhaleTech/CasaOS#165] [Events]
CasaOS has come to version 0.3. The team takes every suggestion and feedback very seriously, and the progress we've made would not have been possible without your participation.

As we combed through your discussions, we found that the level of activity in Discord exceeded our expectations, with shiny ideas and golden words coming out. We also saw a lot of interesting ideas and topics, although they were sometimes repeated a lot. That's why we introduced GitHub Discussions to facilitate better discussion of these ideas.

We've prepared an Ideas incentive event for this purpose. Come and participate and you might win a prize!

### 📅Time
Ideas Discussion: Starting now
Prizes Announcement: May 27, 2022 UTC

### 💡How to participate
1. Read the [Ideas Guidelines](https://github.com/IceWhaleTech/CasaOS/discussions/164) to learn how to participate in the posting of new feature request discussions
2. Post a description of the feature you want in [Ideas](https://github.com/IceWhaleTech/CasaOS/discussions/categories/ideas)
3. Invite your friends to join the discussion and click upvote ⬆️

> Haven't joined our Discord yet? Click [HERE](https://discord.gg/knqAbbBbeX)!

### 🏆Prizes
The CasaOS team will select the following prizes among the Ideas confirmed to be in the development program.
- **Edison Prize**
  - Start an idea discussion
    - Get the Top 1 upvotes
      - Reward: ZimaBoard 832
    - Get the Top 2, 3 upvotes
      - Reward: SanDisk 128GB Extreme Go USB 3.2 Flash Drive
    - Get the Top 4, 5 upvotes
      - Reward: SanDisk 64GB Ultra Fit USB 3.1 Flash Drive
- **Caesar Prize**
  - Leave a comment in the idea discussion
    - Get the Top 1 upvotes
      - Reward: ZimaBoard 832
    - Get the Top 2, 3 upvotes
      - Reward: SanDisk 128GB Extreme Go USB 3.2 Flash Drive
    - Get the Top 4, 5 upvotes
      - Reward: SanDisk 64GB Ultra Fit USB 3.1 Flash Drive
- **Chaplin Prize**
  - Get the Top 1 😄 reaction
    - Reward: SanDisk 64GB Ultra Fit USB 3.1 Flash Drive
- **The Great Majority**
  - Everyone who participated in the discussion and reactions
    - Reward: ZimaBoard Shop 10USD voucher

**Ideas are confirmed to be in the development program based on the following principles**
- Content is carefully written with reference to Ideas guidelines
- There is community consensus and more upvotes
- It is in the team's recent development direction
- The team's recent development resources can meet

**The team's recent development and iteration directions**
- Apps: App Store, browsing, installation, usage of custom apps, etc.
- Files: use, browse and share of Files, etc.
- Storage: Storage space, creation and use of hard disk, etc.

**Comment by JohnGuan** [comment]
### Prizes Announcement

#### Edison Prize
1. @DrMxrcy (https://github.com/DrMxrcy)
2. @AuthorShin (https://github.com/AuthorShin)
3. @Joooost (https://github.com/Joooost)
4. @sio (https://github.com/sio)
5. @bearfrieze (https://github.com/bearfrieze)
6. @steelbucks (https://github.com/steelbucks)

#### Caesar Prize
1. @MX10-AC2N (https://github.com/MX10-AC2N)
2. @zarevskaya (https://github.com/zarevskaya)
3. @Protektor-Desura (https://github.com/Protektor-Desura)
4. @rmcolbert (https://github.com/rmcolbert)
5. @kaylytekjte (https://github.com/kaylytekjte)

#### Chaplin Prize
In the end, no one won this prize, so one more Edison Prize was selected.

#### The Great Majority
- @lemesos (https://github.com/lemesos)
- @BryceLim (https://github.com/BryceLim)
- @CosmoFox (https://github.com/CosmoFox)
- @wolf-yuan-6115 (https://github.com/wolf-yuan-6115)
- @ddelpire (https://github.com/ddelpire)
- @AlexZorzi (https://github.com/AlexZorzi)
- @theeal (https://github.com/theeal)
- @Brottus (https://github.com/Brottus)
- @ghazakhoshmaze (https://github.com/ghazakhoshmaze)
- @Rahmanshaba (https://github.com/Rahmanshaba)
- @Loopringos (https://github.com/Loopringos)
- @Lppftaryjb194 (https://github.com/Lppftaryjb194)
- @Wdnmuy567ja (https://github.com/Wdnmuy567ja)
- @d1y (https://github.com/d1y)
- @rchestradfetorye (https://github.com/rchestradfetorye)
- @eratherargan (https://github.com/eratherargan)
- @RezaRasti79 (https://github.com/RezaRasti79)
- @Tarkarty-borsatentora (https://github.com/Tarkarty-borsatentora)
- @tetradivacopcy (https://github.com/tetradivacopcy)

### How to get the reward?
Using the email address of your GitHub account, send your GitHub and Discord username to my email address: john@icewhale.org.
For the **Edison Prize** and the **Caesar Prize**, please also include your receipt information referring to the following format.
> Name: 
> Address 1: 
> Address 2: 
> City: 
> State: 
> Zip Code: 
> Country: 
> Phone Number:

**Reply by DrMxrcy** [reply]
Congrats Everyone!

**Comment by MX10-AC2N** [comment]
Yesterday, the postwoman delivered a package to me, I had to pay 20€ for delivery taxes but what a wonderful surprise, when I opened the package and discovered the Zimaboard..
## Thanks again to the whole IceWhaleTech team..

**Comment by Protektor-Desura** [comment]
I got my package yesterday. Thank you for the the USB drive.

**Comment by DrMxrcy** [comment]
I got my zimaboard last week. Thanks so much!
